File: go-sdk/pkg/models/operations/skiprevertperiod.go
package operations

type SkipRevertPeriodPathParams struct {
	Database     string `pathParam:"style=simple,explode=false,name=database"`
	Number       string `pathParam:"style=simple,explode=false,name=number"`
	Organization string `pathParam:"style=simple,explode=false,name=organization"`
}

type SkipRevertPeriodRequest struct {
	PathParams SkipRevertPeriodPathParams
}

type SkipRevertPeriodResponse struct {
	ContentType                              string
	StatusCode                               int64
	SkipRevertPeriod200ApplicationJSONObject map[string]map[string]interface{}
}

File: ts-sdk/src/sdk/models/operations/skiprevertperiod.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class SkipRevertPeriodPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=database" })
  database: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=number" })
  number: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=organization" })
  organization: string;
}


export class SkipRevertPeriodRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: SkipRevertPeriodPathParams;
}


export class SkipRevertPeriodResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;

  @SpeakeasyMetadata()
  skipRevertPeriod200ApplicationJSONObject?: Record<string, Record<string, any>>;
}
